### PR TITLE
Migrate tests to JUnit5

### PR DESCRIPTION
**Pull request by Tristan-WorkGH**
Need powsybl/powsybl-parent#65

### CODE REVIEW

File: src/test/java/org/gridsuite/modification/server/modifications/ModificationUtilsTest.java [hunk 1]
```diff
@@ -4,73 +4,57 @@
...
-import static org.assertj.core.api.Assertions.catchRuntimeException;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
 import static org.gridsuite.modification.server.NetworkModificationException.Type.MODIFY_GENERATOR_ERROR;
 
 /**
  * @author David SARTORI <david.sartori_externe at rte-france.com>
  */
-
 class ModificationUtilsTest {
+    private static final ModificationUtils INSTANCE = ModificationUtils.getInstance();
```
Comment (TheMaskedTurtle) [inline src/test/java/org/gridsuite/modification/server/modifications/ModificationUtilsTest.java:26]: Don't refactor that much in hte whole test class

File: src/test/java/org/gridsuite/modification/server/service/NetworkModificationApplicatorTest.java
Comment (TheMaskedTurtle) [inline src/test/java/org/gridsuite/modification/server/service/NetworkModificationApplicatorTest.java:1]: Just put everything back as it was

File: src/test/java/org/gridsuite/modification/server/utils/TestUtils.java [hunk 2]
```diff
@@ -67,26 +66,28 @@ public static void purgeRequests(MockWebServer server) throws UncheckedInterrupt
...
-            destinations.forEach(destination -> assertNull("Should not be any messages in queue " + destination + " : ", output.receive(TIMEOUT, destination)));
-        } catch (NullPointerException e) {
-            // Ignoring
+            for (String destination : destinations) {
+                try {
+                    assertNull(output.receive(TIMEOUT, destination), "Should not be any messages in queue " + destination + " :");
+                } catch (NullPointerException e) {
+                    // Ignoring
+                }
+            }
```
Comment (TheMaskedTurtle) [inline src/test/java/org/gridsuite/modification/server/utils/TestUtils.java:75]: why ?

File: src/test/java/org/gridsuite/modification/server/utils/TestUtils.java [hunk 3]
```diff
@@ -116,7 +117,7 @@ public static void setOperatingStatus(Network network, String identifiableName,
     public static String resourceToString(String resource) throws IOException {
         InputStream inputStream = Objects.requireNonNull(TestUtils.class.getResourceAsStream(resource));
         String content = new String(ByteStreams.toByteArray(inputStream), StandardCharsets.UTF_8);
-        return StringUtils.replaceWhitespaceCharacters(content, "");
+        return content.replaceAll("\\s", "");
```
Comment (TheMaskedTurtle) [inline src/test/java/org/gridsuite/modification/server/utils/TestUtils.java:120]: why ?

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]
You still have some public modifiers in your tests, check sonar

**Review by TheMaskedTurtle — APPROVED** [review]

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**